src/unix/notbsd/android/mod.rs

```diff
@@ -727,74 +712,6 @@ pub const NLA_TYPE_MASK: ::c_int = !(NLA_F_NESTED | NLA_F_NET_BYTEORDER);
 pub const SIGEV_THREAD_ID: ::c_int = 4;
 
 f! {
-    pub fn sigemptyset(set: *mut sigset_t) -> ::c_int {
```

Comment:
Ah ok, in that case sounds great to me!

Android-9 is such an ancient NDK that it seems fine to not require support for that here. If project want that level of compatibility (e.g. libstd) we can vendor the impls ourselves.